
From: Andrew Morton <akpm@osdl.org>
Subject: aic79xx need to select SCSI_TRANSPORT_SPI

Against git-scsi-misc:

drivers/built-in.o(.text+0xdcccd): In function `ahd_linux_slave_configure':
drivers/scsi/aic7xxx/aic79xx_osm.c:636: undefined reference to `spi_dv_device'
drivers/built-in.o(.text+0xdd7b1): In function `ahd_send_async':
drivers/scsi/aic7xxx/aic79xx_osm.c:1652: undefined reference to `spi_display_xfer_agreement'
drivers/built-in.o(.init.text+0x7b4d): In function `ahd_linux_init':
drivers/scsi/aic7xxx/aic79xx_osm.c:2765: undefined reference to `spi_attach_transport'
drivers/built-in.o(.init.text+0x7c94):drivers/scsi/aic7xxx/aic79xx_osm.c:2774: undefined reference to `spi_release_transport'
drivers/built-in.o(.exit.text+0x72c): In function `ahd_linux_exit':
drivers/scsi/aic7xxx/aic79xx_osm.c:2783: undefined reference to `spi_release_transport'

Cc: James Bottomley <James.Bottomley@steeleye.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---

 drivers/scsi/aic7xxx/Kconfig.aic79xx |    1 +
 1 files changed, 1 insertion(+)

diff -puN drivers/scsi/aic7xxx/Kconfig.aic79xx~aic79xx-needs-scsi_transport_spi drivers/scsi/aic7xxx/Kconfig.aic79xx
--- 25/drivers/scsi/aic7xxx/Kconfig.aic79xx~aic79xx-needs-scsi_transport_spi	2005-08-06 22:34:01.000000000 -0700
+++ 25-akpm/drivers/scsi/aic7xxx/Kconfig.aic79xx	2005-08-06 22:34:22.000000000 -0700
@@ -5,6 +5,7 @@
 config SCSI_AIC79XX
 	tristate "Adaptec AIC79xx U320 support"
 	depends on PCI && SCSI
+	select SCSI_SPI_ATTRS
 	help
 	This driver supports all of Adaptec's Ultra 320 PCI-X
 	based SCSI controllers.
_
